fr-portfolio.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+</head>
+<body>	
+	<header class="navbar">
+		<a href="#portfolio">Portfolio</a>
+		<a href="#about">About</a>
+		<a href="#contact">Contact</a>
+	</header>
+	<section class="welcome-banner" >
+							<h1>Freddie's Portfolio</h1>
+										
+	</section>	
+	<section id="about" class="about-alt">
+		<h2> About me </h2>
+			<div class="about-me">
+				<div>
+					<img src="https://via.placeholder.com/350x200">
+				</div>
+				<div>
+					<p>Hi this is where I should put some information about myself</p>
+				</div>
+			</div>		
+	</section>
+		
+	<section id="portfolio" class="projects">
+		<h2>Portfolio</h2>
+			<div class="portfolio-container">
+				<div>
+					<img src="https://via.placeholder.com/350x200">
+					<p>This is where my project information will go</p>
+				</div>
```

Comment:
The `<figure>` tag is great for wrapping up media content with a caption: e.g 
`<figure>`
    `<img src="etc>`
    `<figcaption>`Caption for picture `</figcaption>`
 `<figure>`
 
 It ensures a caption will stay with the images it is meant to be attached to, even if you move them about in CSS